public/vendor/ansicolor/ansicolor.ts

```diff
@@ -62,7 +62,9 @@ const clean = (obj: any) => {
 class Color {
   background: boolean | undefined;
   name: string | undefined;
-  brightness: number | undefined;
+  background?: boolean;
+  name?: string;
+  brightness?: number;
```

Comment:
you have duplicate properties now (background & name)

Reply:
Fixed it yesterday, but forgot to push the changes